packages/prci/src/StatusView.ts

```diff
@@ -100,8 +100,15 @@ export class StatusView extends LitElement {
         );
       }
 
-      // If no more demands are processing we can stop reloading this request
-      if (this._processingDemands.length === 0 && this._intervalId) {
+      // If no more demands are processing, the reload interval exists, and the data for
+      // all ACCESS responses has arrived, stop reloading the request.
+      if (
+        this._processingDemands.length === 0 &&
+        this._intervalId &&
+        !this._completedDemands.some(
+          d => d.requested_action === ACTION.ACCESS && !d.data
```

Comment:
(actually on line 116, so to solve later on)

Making a request every 3 seconds is a very bad approach here IMHO. It won't scale. Remember you should always try to make as few requests as possible. This request should be triggered by the user, directly (i.e. reload button) or indirectly (i.e. loading the view) to determine with @Clementinev. Alternatively, you could use a web socket if really necessary, but I don't think it is here (to discuss with @m4rk055).